project-service/api.md

```diff
@@ -0,0 +1,594 @@
+# Project Service External API
+
+## Users
+
+### Fetch Users
+
+Returns json data of all users as an array
+
+- URL
+  + `/users`
```

Comment:
I see. That will be useful.

But this request shouldn't return all users; it should probably return a small subset of users that match the current input data (for autocomplete, if we manage to pull this off).